README.md

```diff
@@ -42,12 +42,12 @@ cmake ..
 make -j
 ```
 
-If you don't have MPI, you can disable MPI dependency using the CMake option `-DENABLE_MPI=OFF`:
+If you don't have MPI, you can disable MPI dependency using the CMake option `-DCORENRN_ENABLE_MPI=OFF`:
```

Comment:
I was thinking if the `CORENRN_<option>` prefix is needed to be added in the various options. It doesn't seem to me very practical for the user to have to write this prefix multiple times for generic options. But it just might me, so feel free to keep it :smile:

Reply:
the problem is when we use CoreNEURON with NRN as submodule, you get all CMake options mixed. And hence you need a way to distinguish those (and this is commonly done).